_locales/ar/messages.json

```diff
@@ -144,15 +144,15 @@
     "description": "Button text; Create filter dialog; prefs/filters"
   },
   "custom_filename": {
-    "message": "خصّص اسم الملف",
+    "message": "اسم ملف مخصص",
     "description": "Label text; single window"
   },
   "deffilter_all": {
     "message": "كل الملفات",
     "description": "Filter label for the All files filter"
   },
   "deffilter_arch": {
-    "message": "الأرشيفات (zip،‏ rar،‏ 7z، …)",
+    "message": "الملفات المضغوطة (zip،‏ rar،‏ 7z، …)",
```

Comment:
Archive is a very known word you see even in news. And why translating one word into two if we have the shorter?

Reply:
And, not all archives are compressed files. You can just use it to contain several files with no compression at all.